HCK01_2022_Virtual/README.md

```diff
@@ -121,9 +238,28 @@ In future hackathons, we will focus on:
 
 [How to add a new reproducible registration processing pipeline?](./ReproducibleResource/README.md)
 
+### Challenge Dataset 1: fMOST Mouse Brain Registration to CCFv3
+
+The fMOST brain volumes and CCFv3 atlas for the hackathon are
+available [on the BIL here](https://download.brainlib.org/hackathon/2022_GYBS/data/). The input is a
+single fMOST NIFTI brain volume.
+
+### Challenge Dataset 1: Light-Sheet Imaged Mouse Brain Registration to CCFv3
+
+The Light-sheet imaged brain volumes are available
+[on the BIL here](https://download.brainlib.org/hackathon/2022_GYBS/lightsheet/). For this challenge
+set, the data and atlas is provided for a single brain hemisphere (for size considerations and to
+mimic most real world experiments). Multiple data replicates are provided. We provide one
+dataset `subject0.zarr` in full imaging resolution for those seeking a bigger challenge. As a
+convenience we provide the challenge sets at approximately atlas size as
+Nifti(`subjectN_25.nii. gz`)
```

Comment:
`.nii. gz` -> `.nii.gz`

Reply:
resolved in https://github.com/InsightSoftwareConsortium/GetYourBrainStraight/commit/fc96c8c7852339bc94e75f55912e2af4ef33702b